libs/dh/metering-point/feature-identity-and-master-data/src/lib/identity/dh-metering-point-identity.component.html

```diff
@@ -14,42 +14,84 @@
 See the License for the specific language governing permissions and
 limitations under the License.
 -->
-<div
-  class="overview__header"
-  *transloco="let t; read: 'meteringPoint.overview'"
->
-  <div class="watt-text-s">
-    <span [title]="t('meteringPointType')">{{
-      translationKeys?.meteringPointType | transloco | dhEmDashFallback
-    }}</span>
-
-    (<span [title]="t('meteringMethod')">{{
-      translationKeys?.meteringMethod | transloco | dhEmDashFallback
-    }}</span
-    >)
-  </div>
-
-  <h1 [title]="t('gsrnNumber')">{{ meteringPoint?.gsrnNumber }}</h1>
-
-  <div class="watt-text-s">
-    <span [title]="t('connectionState')">
+<div class="overview-header" *transloco="let t; read: 'meteringPoint.overview'">
+  <div class="watt-text-s overview-header__row">
+    <span [title]="t('connectionState')" class="watt-space-inline-xs">
       <dh-status-badge
-        [connectionState]="meteringPoint?.connectionState"
+        [connectionState]="identityDetails.connectionState"
       ></dh-status-badge>
     </span>
 
-    <ng-container *ngIf="meteringPoint?.settlementMethod">
-      |
-      <span [title]="t('settlementMethod')">
-        {{ translationKeys?.settlementMethod | transloco }}
-      </span>
-    </ng-container>
+    <span [title]="t('meteringMethod')">
+      {{ translationKeys?.meteringPointType | transloco | dhEmDashFallback }}
+      ({{ translationKeys?.meteringMethod | transloco | dhEmDashFallback }}) |
+    </span>
+
+    <span
+      *ngIf="identityDetails?.settlementMethod"
+      [title]="t('settlementMethod')"
+    >
+      {{ translationKeys?.settlementMethod | transloco }} |
+    </span>
 
-    <ng-container *ngIf="meteringPoint?.readingOccurrence">
+    <span [title]="t('readingOccurrence')">
+      {{ translationKeys?.readingOccurrence | transloco | dhEmDashFallback }}
       |
```

Comment:
I don't understand why it adds a `|` ?

Reply:
The translation doesn't add a `|`. The issue is that hovering on the `|` symbol after the text, will also trigger the `title` tooltip to appear (because the `|` is inside the `span` element where `title` is applied). My suggestion was to move the `|` outside the `span`, so it's treated as normal text.